test/Rings-test.jl

```diff
@@ -67,3 +67,14 @@ end
   @test is_finite(GF(2))
   @test !is_finite(QQ)
 end
+
+import LinearAlgebra: dot
+
+@testset "dot" begin
```

Comment:
```suggestion
@testset "dot" begin
  dot = AbstractAlgebra.LinearAlgebra.dot

```
better import in a scope to not make `dot` available